README.md

```diff
@@ -10,6 +10,10 @@ Time-handling functionality from netcdf4-python
 [![Commits Status](https://img.shields.io/github/commits-since/UniData/cftime/latest.svg)](https://github.com/UniData/cftime/commits/master)
 
 ## News
+12/05/2018:  version 1.0.3.1 released.  Bugfix release (fixed issue with installation
```

Comment:
It's hilariously ironic that your American dates are just plain wrong i.e. not British :stuck_out_tongue_winking_eye: